build-tools/packages/build-cli/docs/generate.md

```diff
@@ -9,6 +9,7 @@ Generate commands are used to create/update code, docs, readmes, etc.
 * [`flub generate changelog`](#flub-generate-changelog)
 * [`flub generate changeset`](#flub-generate-changeset)
 * [`flub generate entrypoints`](#flub-generate-entrypoints)
+* [`flub generate layerGeneration`](#flub-generate-layergeneration)
```

Comment:
Is there any way we could add something to clarify the generation in this context? Maybe `compatLayerGeneration`?